src/zcash/address/unified.h

```diff
@@ -5,27 +5,23 @@
 #ifndef ZCASH_ZCASH_ADDRESS_UNIFIED_H
 #define ZCASH_ZCASH_ADDRESS_UNIFIED_H
 
-#include "transparent.h"
 #include "key_constants.h"
 #include "script/script.h"
+#include "transparent.h"
 #include "zcash/address/orchard.hpp"
 #include "zip32.h"
 
 #include <variant>
 
-namespace libzcash {
+namespace libzcash
+{
 
 // prototypes for the classes handling ZIP-316 encoding (in Address.hpp)
 // TODO: ZIP-316 encoding should probably be moved here
 class UnifiedAddress;
 class UnifiedFullViewingKey;
 
-enum class ReceiverType: uint32_t {
-    P2PKH = 0x00,
-    P2SH = 0x01,
-    Sapling = 0x02,
-    Orchard = 0x03
-};
+enum class ReceiverType : uint32_t { P2PKH = 0x00, P2SH = 0x01, Sapling = 0x02, Orchard = 0x03 };
```

Comment:
Hmm, I'd prefer `enum` items always on separate lines. I think that's `AllowShortEnumsOnASingleLine: false](https://clang.llvm.org/docs/ClangFormatStyleOptions.html#allowshortenumsonasingleline).